examples/va/nvidia-mdev/control-plane.md

```diff
@@ -0,0 +1,52 @@
+# Configuring networking and deploy the OpenStack control plane
+
+## Assumptions
+
+- A storage class called `local-storage` should already exist.
+
+## Initialize
+
+Switch to the "openstack" namespace
+```
+oc project openstack
+```
+Change to the nvidia-mdev directory
```

Comment:
It looks like the `cd ...` command example was inadvertently removed

Reply:
Huh I tried to use the GUI commit but I guess it just deleted it.  Let me add in your recommended change manually.